### ISSUE TITLE
Mac Trackpad!

### ISSUE THREAD

**Issue by shayneoneill** [armory3d/armorpaint#1392]
Ok this is definately a stupid newbie question BUT...... How do we do middle click actions on a Mac Trackpad? The instructions all talk about using the middle click to navigate, but we dont actually have a middle click, which means I cant actually navigate the viewport. Obviously there is a way, being that the mac versions for sale(!!!) but I cant find it anywhere in the documentation.

Where am I going wrong here?

**Comment by rpaladin** [comment]
You can assign a different button or key for the middle-mouse function I believe by going to your ArmorPaint keymap preferences.
Preferences: `Menu bar - Edit - Preferences... - Keymap`.
Reference: <https://armorpaint.org/manual>

![image](https://user-images.githubusercontent.com/69180012/166400123-7c9aae64-f189-4df9-9410-ae871fb9194b.png)

**Comment by shayneoneill** [comment]
Yeah I saw that. Unfortunately the + key doesnt seem to work in those inputs which makes changing those keys a bit tricky.

**Comment by MathemanFlo** [comment]
As a temporary workaround you could edit the keymap directly. Locate the ArmorPaint executable. Go to `data/keymap_presets` and edit your chosen keymap directly.
